docs/creating-studies.md

```diff
@@ -24,15 +24,21 @@ to any build/export call to tell it where to look for your work.
 
 ## Creating a new study
 
-The easiest way to get started with a new study is to use `cumulus-library` to
-create a manifest for you. You can do this with by running:
+There are two ways to get started with a new study:
+
+1. Use `cumulus-library` to create a manifest for you. You can do this with by running:
 ```bash
 cumulus-library create ./path/to/your/study/dir
 ```
-We'll create that folder if it doesn't already exist. We recommend you use a name
-relevant to your study (we'll use `my_study` forthis document). The folder name is
-the same thing you will use as a target with `cumulus_library` to run your study's
-queries.
+We'll create that folder if it doesn't already exist. 
+
+2. Fork the [
+Cumulus library template repo](https://github.com/smart-on-fhir/cumulus-library-template),
+renaming your fork, and cloning it directly from github.
```

Comment:
I originally only provided CLI creation but git based was a specific request.

I think it's generally a case of 'which is a user going to find more painful'? A CLI can be intimidating, but so can git - it depends on your background. If I was going to remove one at this point it would be the CLI version, since it's not being used currently except by me.